src/modules/services/submodule/voting/components/CalanderInput.vue

```diff
@@ -57,12 +57,12 @@ export default{
 </script>
 <style lang="scss">
 
-.p-inputtext{
-  border: 0px;
-  padding: 0;
-  font-size: 14px;
-  text-align: right;
-}
+// .p-inputtext{
```

Comment:
if comments are no longer relevant, please remove

Reply:
noted, will remove it